src/backend/distributed/utils/citus_clauses.c

```diff
@@ -452,3 +461,40 @@ CitusIsMutableFunction(Node *node)
 
 	return false;
 }
+
+
+/* FixFunctionArguments applies expand_function_arguments to all function calls. */
+static void
+FixFunctionArguments(Node *expr)
+{
+	FixFunctionArgumentsWalker(expr, NULL);
+}
+
+
+/* FixFunctionArgumentsWalker is the helper function for fix_funcargs. */
+static bool
+FixFunctionArgumentsWalker(Node *expr, void *context)
+{
+	if (expr == NULL)
+	{
+		return false;
+	}
+
+	if (IsA(expr, FuncExpr))
+	{
+		FuncExpr *funcExpr = castNode(FuncExpr, expr);
+		HeapTuple func_tuple =
```

Comment:
For other readers:

https://github.com/postgres/postgres/blob/master/src/backend/optimizer/util/clauses.c#L2427-L2434